draft-ietf-ppm-dap.md

```diff
@@ -413,6 +413,10 @@ Batch:
 : A set of reports (i.e., measurements) that are aggregated into an aggregate
   result. As defined in {{!VDAF}}.
 
+Batch bucket:
+: Data associated with a given batch allowing the aggregators to perform
```

Comment:
* We should try to make it clear here that there is a many-to-one relationship between buckets and batches.